classes/local/renderer.php

```diff
@@ -60,6 +60,12 @@ public function get_data(): \stdClass {
             $data->$name = $ds->get_data();
         }
 
+        // Create a debug variable that contains the whole structure.
+        if ($CFG->debugdisplay) {
```

Comment:
I'd wouldn't want this tangled up in CFG because I will 100% want this on in a prod site testing a new template and we don't want that on in prod

Reply:
Instead make this its own pseudo data source so we can just do {{{debug}}} and it will output the whole chunk, and I would also put this into the template as the default so you get something for a brand new template and keep tweaking from there

Reply:
Do you want another switch, or should I make it always included?

Reply:
Have it always included. If you don't want it shown then you can just remove {{{debug}}} from the template, or convert it to a mustache comment so you can add it back later easilt

Reply:
It doens't need to be a full data source if that is a lot of boilerplate for not a heap of value